src/quota/azext_quota/generated/_help.py

```diff
@@ -0,0 +1,216 @@
+ineLearningServices/locations/eastus"
+"""
+
+helps['quota usage show'] = """
+    type: command
+    short-summary: "Get the current usage of a resource."
+    examples:
+      - name: Quotas_UsagesRequest_ForCompute
+        text: |-
+               az quota usage show --resource-name "standardNDSFamily" --scope "subscriptions/00000000-0000-0000-0000-0\
+00000000000/providers/Microsoft.Compute/locations/eastus"
+      - name: Quotas_UsagesRequest_ForNetwork
+        text: |-
+               az quota usage show --resource-name "MinPublicIpInterNetworkPrefixLength" --scope \
+"subscriptions/00000000-0000-0000-0000-000000000000/providers/Microsoft.Network/locations/eastus"
+"""
+
+helps['quota list'] = """
+    type: command
+    short-summary: "Get a list of current quota limits of all resources for the specified scope. The response from \
+this GET operation can be leveraged to submit requests to update a quota."
+    examples:
+      - name: Quotas_listQuotaLimitsForCompute
+        text: |-
+               az quota list --scope "subscriptions/00000000-0000-0000-0000-000000000000/providers/Microsoft.Compute/lo\
+cations/eastus"
+      - name: Quotas_listQuotaLimitsForNetwork
+        text: |-
+               az quota list --scope "subscriptions/00000000-0000-0000-0000-000000000000/providers/Microsoft.Network/lo\
+cations/eastus"
+      - name: Quotas_listQuotaLimitsMachineLearningServices
+        text: |-
+               az quota list --scope "subscriptions/00000000-0000-0000-0000-000000000000/providers/Microsoft.MachineLea\
+rningServices/locations/eastus"
+"""
+
+helps['quota show'] = """
+    type: command
+    short-summary: "Get the quota limit of a resource. The response can be used to determine the remaining quota to \
+calculate a new quota limit that can be submitted with a PUT request."
+    examples:
+      - name: Quotas_Get_Request_ForCompute
+        text: |-
+               az quota show --resource-name "standardNDSFamily" --scope "subscriptions/00000000-0000-0000-0000-0000000\
+00000/providers/Microsoft.Compute/locations/eastus"
+      - name: Quotas_UsagesRequest_ForNetwork
+        text: |-
+               az quota show --resource-name "MinPublicIpInterNetworkPrefixLength" --scope \
+"subscriptions/00000000-0000-0000-0000-000000000000/providers/Microsoft.Network/locations/eastus"
+"""
+
+helps['quota create'] = """
+    type: command
+    short-summary: "Create the quota limit for the specified resource with the requested value. To update the quota, \
+follow these steps: 1. Use the GET operation for quotas and usages to determine how much quota remains for the \
+specific resource and to calculate the new quota limit. These steps are detailed in [this \
+example](https://techcommunity.microsoft.com/t5/azure-governance-and-management/using-the-new-quota-rest-api/ba-p/21836\
+70). 2. Use this PUT operation to update the quota limit. Please check the URI in location header for the detailed \
+status of the request."
```

Comment:
Please overwrite this help in manual folder. The short-summary is too long.

```suggestion
    short-summary: "Create the quota limit for the specified resource with the requested value."
```

Reply:
It has been modified.